tests/tests_e3/fs/main_test.py

```diff
@@ -208,10 +208,19 @@ def test_tree_state():
     state6 = e3.fs.get_filetree_state("toto")
     assert isinstance(state6, str)
 
+    with open("toto", "wb") as f:
+        f.write(b"hello world.")
+
     # check that get_filetree_state accept unicode
     state7 = e3.fs.get_filetree_state("toto")
     assert isinstance(state7, str)
 
+    # check that get_filetree_state with hash_content is different that the
+    # previous state
+    state8 = e3.fs.get_filetree_state("toto", hash_content=True)
+    assert isinstance(state8, str)
+    assert state8 != state7
```

Comment:
As the function behavior depends on the path type (directory vs file) should a test over a directory be added too ?

Reply:
Done